cypress/integration/UserGroups/UserGroupObject.ts

```diff
@@ -0,0 +1,83 @@
+const BASE_URL = process.env.BASE_URL || `http://localhost:${process.env.PORT || 8000}`;
+const hash = Cypress.env('router') === 'hash' ? '#/' : '';
+
+export const selectors = {
```

Comment:
move this file elsewhere as it is being called as test 

https://github.com/EscolaLMS/Admin/actions/runs/1516827587 see videos